example/src/androidTest/java/org/wordpress/android/fluxc/mocked/MockedStack_WCProductsTest.kt

```
@@ -29,7 +30,7 @@ class MockedStack_WCProductsTest : MockedStack_Base() {
     private var lastAction: Action<*>? = null
     private var countDownLatch: CountDownLatch by notNull()
 
-    private val remoteProductId = 1537L
+    private val remoteProductId = BuildConfig.TEST_WC_PRODUCT_ID.toLong()
```

@nbradbury I think you mean to change this in `ReleaseStack_WCProductTest` rather than `MockedStack_WCProductsTest` - this is also the reason Travis is failing.

---

Of course :( Fixed in 055a575